nexus/db-model/src/deployment.rs

```diff
@@ -1217,6 +1218,53 @@ impl TryFrom<DbBpZoneImageSourceColumns> for BlueprintZoneImageSource {
     }
 }
 
+#[derive(Queryable, Clone, Debug, Selectable, Insertable)]
+#[diesel(table_name = bp_single_measurements)]
+//#[diesel(check_for_backend(diesel::pg::Pg))]
+pub struct BpSingleMeasurement {
+    pub blueprint_id: DbTypedUuid<BlueprintKind>,
+    pub sled_id: DbTypedUuid<SledKind>,
+    pub id: DbTypedUuid<PhysicalDiskKind>,
+
+    pub image_artifact_sha256: Option<ArtifactHash>,
+    pub prune: bool,
+}
+
+impl BpSingleMeasurement {
+    pub fn new(
+        blueprint_id: BlueprintUuid,
+        sled_id: SledUuid,
+        measurement: &BlueprintSingleMeasurement,
+    ) -> Self {
+        Self {
+            blueprint_id: blueprint_id.into(),
+            sled_id: sled_id.into(),
+            // XXX lol this should probably just be a regular uuid
```

Comment:
Ah, I see that you called the field `id` above because it isn't actually a disk id. It might be worth adding a new `MeasurementUuid` type to `omicron_uuid_kinds`. I'm not sure when we've decided to do that. CC @sunshowers.

Reply:
yeah you caught one of my hacks catching up with me. I can just make a `MeasurementUuid` .